### DISCUSSION TITLE
turborepo & remix - functions missing dependencies

### DISCUSSION THREAD

**Discussion by marcolink** [vercel/turborepo#496] [Help] [answered]
Hi,

I have set up a vanilla `turborepo` project, that holds a vanilla remix project under `apps/my-remix-app`.  
The remix project has been create with `npx create-remix@latest` and `vercel` as "server". 

When building it on vercel, the build succeeds, but when running it, the function throws

```
2021-12-28T20:52:42.401Z	undefined	ERROR	Cannot find module '@remix-run/vercel'
```

In my understanding, the `api/index.js` file ... 

```javascript
const { createRequestHandler } = require("@remix-run/vercel");

module.exports = createRequestHandler({
  build: require("./_build")
});
```
...  is not bundled in any way, and can't resolve hoisted dependencies. 

> Any thoughts on how to solve this?

I made an example project - feel free to clone https://github.com/marcolink/turbo-remix

**Comment by ShubhamVerma1811** [comment]
Were you able to get it to work?
Any updates on this?

**Reply by okonet** [reply, marked as answer]
Yes! Check this https://github.com/webstudio-is/webstudio-designer/pull/123 out
